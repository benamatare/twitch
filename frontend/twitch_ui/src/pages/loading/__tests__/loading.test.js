import React from "react";
import Loading from '../components/loading';

it('renders', () => {
    const wrapper = shallow(
        <Loading />
    );
    expect(wrapper).toMatchSnapshot();
})
