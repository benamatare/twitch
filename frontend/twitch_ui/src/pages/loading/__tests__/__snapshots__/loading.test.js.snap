// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="loading-screen-parent"
>
  <Spinner
    fadeIn="full"
    name="wave"
    noFadeIn={false}
    overrideSpinnerClassName=""
  />
</div>
`;
